openebs/openebs-provisioner_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseClassParameters(t *testing.T) {
```

Comment:
These tests can be simplified (remove the duplicate code) by following the TDD approach.

Reply:
Done, changed to TDD approach.